psychopy/tests/test_sound/test_multichannel_asio.py

```diff
@@ -1,8 +1,7 @@
-from psychopy import prefs
-prefs.general['audioLib'] = ['sounddevice']
```

Comment:
This actually needs to be here - in this test we're deliberately changing the sound lib in preferences which needs to be done before importing sound (otherwise the sound will be imported thinking it's a different preferred lib) so on this occasion you need to just override PEP8. You might be able to do `# noqa` as a comment on that line to stop you editor complaining about it

Reply:
Thanks, I have now read some of the documentation. I performed a new pull request on dev, with the changes you suggested. Thanks!